airbyte-commons-worker/src/main/java/io/airbyte/workers/process/KubePodProcess.java

```diff
@@ -109,9 +109,14 @@ public class KubePodProcess extends Process implements KubePod {
   private static final String DEFAULT_MEMORY_LIMIT = "50Mi";
   private static final String DEFAULT_CPU_REQUEST = "0.1";
   private static final String DEFAULT_CPU_LIMIT = "0.2";
+  private static final String DEFAULT_SOCAT_CPU_REQUEST = "1.0";
+  private static final String DEFAULT_SOCAT_CPU_LIMIT = "1.5";
```

Comment:
we probably need to provide a way to override these via env vars since these are high for our dev envs and also high for a portion of our OSS users - it would also make it easier for us to test different configurations and the performance effect.